tchannel/thrift/client.py

```diff
@@ -75,15 +78,21 @@ def post_comment(articleId, msg, hostport=None):
 
     method_names = get_service_methods(service_module.Iface)
 
-    def new(cls, tchannel, hostport=None, trace=False):
+    def new(cls, tchannel, hostport=None, trace=False, protocol_headers=None):
```

Comment:
Seems like we should add it now, so that both async and sync get trace support. What do you think @abhinav @blampe ?